sonar-java-plugin/src/main/java/org/sonar/plugins/java/bridges/DesignBridge.java

```diff
@@ -87,7 +87,7 @@ public void saveDesign(Project sonarProject) {
     Measure dsmMeasure = new Measure(CoreMetrics.DEPENDENCY_MATRIX, dsmJson).setPersistenceMode(PersistenceMode.DATABASE);
     context.saveMeasure(sonarProject, dsmMeasure);
 
-    profiler.stop();
+    profiler.stopInfo();
 
     for (Resource sonarPackage : directories) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Resource"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)